README.md

```diff
@@ -109,60 +120,244 @@ Super similar to `compare_relations`, except it takes two select statements. Thi
+* `exclude_columns` (optional): Any columns you wish to exclude from the validation.
+
+### Usage:
+```sql
+
+{% set old_relation = adapter.get_relation(
+      database = "old_database",
+      schema = "old_schema",
+      identifier = "fct_orders"
+) -%}
+
+{% set dbt_relation = ref('fct_orders') %}
+
+{{ audit_helper.compare_which_columns_differ(
+    a_relation = old_relation,
+    b_relation = dbt_relation,
+    exclude_columns = ["loaded_at"],
+    primary_key = "order_id"
+) }}
+
+```
+
+## compare_relation_columns ([source](macros/compare_relation_columns.sql))
+This macro generates SQL that can be used to compare the schema (ordinal position and data types of columns) of two relations. This is especially useful when:
+* Comparing a new version of a relation with an old one, to make sure that the structure is the same
+* Helping figure out why a `union` of two relations won't work (often because the data types are different)
+
+### Output:
+| column_name | a_ordinal_position | b_ordinal_position | a_data_type       | b_data_type       | has_ordinal_position_match | has_data_type_match | in_a_only | in_b_only | in_both |
+|-------------|--------------------|--------------------|-------------------|-------------------| -------------------------- | ------------------- | --------- | --------- | ------- |
+| order_id    | 1                  | 1                  | integer           | integer           |                       True |                True |     False |     False |    True |
+| customer_id | 2                  | 2                  | integer           | integer           |                       True |                True |     False |     False |    True |
+| order_date  | 3                  | 3                  | timestamp         | date              |                       True |               False |     False |     False |    True |
+| status      | 4                  | 5                  | character varying | character varying |                      False |                True |     False |     False |    True |
+| amount      | 5                  | 4                  | bigint            | bigint            |                      False |                True |     False |     False |    True |
+
+Note: For adapters other than BigQuery, Postgres, Redshift, and Snowflake, the ordinal position is inferred based on the response from dbt Core's `adapter.get_columns_in_relation()`, as opposed to being loaded from the information schema.
+
+### Arguments:
+* `a_relation` and `b_relation`: The [relations](https://docs.getdbt.com/reference/dbt-classes#relation) you want to compare.
+
+### Usage:
+```sql
+
+{% set old_relation = adapter.get_relation(
+      database = "old_database",
+      schema = "old_schema",
+      identifier = "fct_orders"
+) -%}
+
+{% set dbt_relation = ref('fct_orders') %}
+
+{{ audit_helper.compare_relation_columns(
+    a_relation=old_relation,
+    b_relation=dbt_relation
+) }}
+
+```
+
+# Print Output To Logs
```

Comment:
```suggestion
## Advanced Usage 
### Print Output To Logs
```

Reply:
I'm all for this, but we should add to the table of contents as well!